app/controllers/v0/user/mhv_user_accounts_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
       private
 
       def set_mhv_user_account
-        @mhv_user_account = current_user.mhv_user_account
+        @mhv_user_account = current_user.mhv_user_account(break_cache: true)
```

Comment:
I think we can set this up to call the `MHV::UserAccount::Creator` directly. Because I think we don't need to hide this behind the facility ids check. If a FE is explicitly calling for this then I think it's safe to call on the user regardless of whether or not they're MHV eligible